anti-analysis/anti-forensic/hide-shadow-copy.yml

```diff
@@ -0,0 +1,24 @@
+rule:
+  meta:
+    name: hide shadow copy
+    namespace: anti-analysis/anti-forensic
+    authors:
+      - blas.kojusner@mandiant.com
+    scope: function
+    references:
+      - https://github.com/gtworek/PSBits/tree/master/HideSnapshot
+    examples:
+      - 30EE31A532545D2FB456B6204DC443D5:0x140001070
+  features:
+    - and:
+      - or:
+        - and:
+          - description: Volume Snapshot type GUID for hidden
+          - number: 0xF12142B4 = GUID slice 0
+          - number: 0x49AF9A4B = GUID slice 1
+          - number: 0x0C7051A8 = GUID slice 2
+          - number: 0xBEC2FD42 = GUID slice 3
+        - bytes: B4 42 21 F1 4B 9A AF 49 A8 51 70 0C 42 FD C2 BE = GUID
```

Comment:
can you include the rendered GUID here in the description

Reply:
also, would you add a string feature like `string: 00000-0000-00-00fdfdsafdafdsa` to capture that (uncommon) encoding mechanism?